examples/rstudio/test_rstudio.py

```diff
@@ -0,0 +1,33 @@
+import getpass
```

Comment:
Is there a reason not to make this script executable with a shebang line?

Also it could use some comments to explain what is going on. There's nothing else like it in the Charliecloud test suite, so the audience won't be familiar.